meetings/2025-05-28.md

```diff
@@ -0,0 +1,128 @@
+
+
+### [JW] Providing Lightweight infrastructure for publishing Vocabs
+
+* JW: Proposal is to have a repo for vocabs with governance similar to definitelytyped (e.g., if you are first to contribute the vocab, then you're the one responsible for reviewing feedback with core maintainers), and for us to ask w3c to publish vocab from under their name space (e.g., w3c.org/solid/recipes)
+* JW: There is a grant proposal to NLNet that was made to do a similar thing for Shapes
+* JW: We had discussion last week, Michiel wanted to be able publish vocab in 20 min. Extending conversation from solid/vocab issues hanging unresolved.
+* JW: There has been discussion on the mailing list, around URN, w3id.org, etc. I had a quick conversation with Tim; we can create definitelytyped-style repository. Anyone would be able to create a PR that creates a new vocab document. When you create a new vocab, you become owner of that vocab and responsible for creating subsequent PRs. Once they are merged into that repo, they would be published in W3C namespace, e.g., w3c.org/ns/solid/recipes 
+* JW: We would use it as a way to help us align on the vocabs.
+* eP: I would need to look up the last conversation. I don't think I want to publish terms that may be thrown out in the w3c namespace. There is a risk of getting duplicates, like multiple vocabs for calendars. Is it something for long term, or something that will be thrown away?
+* JW: It is not intended for terms that people will throw away; it is meant for long-lived vocabs. One practice could be that once you start creating an app, you would create a recipes vocab document, and put any terms there that you already stablized. When you still have early-stage term, you would wait to publish it.
+* eP: Who makes the decision whether something is stable?
+* JW: You pretty much trust people. I don't think there will be an issue with creating multiple vocabs. We would create statistics on which vocabs are being used by which apps. 
+* eP: I would trust people's intentions, but be careful with their semweb knowledge.
+* JW: We could create a basic set of guardrails. I agree it doesn't solve all the issues; ontology creation is complex. We could look at having maybe one or two maintainers, possibly with support of ODI.
+* HZ: The enthusiast in me says great. I'm worried about using W3C namespace. People may get the wrong impression about stability based on that. I'm not sure what the *solid* vocabulary means. Do we have infrastructure for it? Do we have a stamp of aproval?
+* JW: To respond about W3C namespace, one potential mitigation would be to have a clear warning. W3C could say no, anyway. It would be useful to have a stable home. I was proposing w3id.org, and Tim suggested W3C instead. 
```

Comment:
```suggestion
* JW: To respond about W3C namespace, one potential mitigation would be to have a clear warning. W3C could say no, anyway. It would be useful to have a stable home. I was proposing w3id.org, and Tim suggested W3C instead.
* [TT (after meeting): One advantage of w3id.org is that it's strictly a redirect service, not an actual host environment. Vocabs could change their actual deployment host without breaking any active use. W3C seems an odd host for non-W3C projects, even if Solid sprang from there.]
```